package.json

```diff
@@ -135,6 +135,9 @@
   "optionalDependencies": {
     "appdmg": "^0.6.4"
   },
+  "resolutions": {
+    "jquery": "3.6.3"
+  },
```

Comment:
This is to prevent clash of different jquery versions when plugins are loaded in.